imagespace/web_external/templates/widgets/imageDetailWidget.jade

```diff
@@ -34,6 +34,15 @@
                   b= key
                 dd
                   span !{val}
+        if facetviewAdsUrl && image.has('relevantAds') && image.get('relevantAds').length
+          h4 Ads
+          dl.dl-horizontal
+            for ad_id in image.get('relevantAds')
+              dt
+
+              dd
+                span
+                  a(href=facetviewAdsUrl + '?query=resourcename_t_md:' + ad_id) !{ad_id.substr(1, 40)}...
```

Comment:
I think you might need to enclose the ad_id in quotes.  If you've tested this and it works, ignore me here.